<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>809</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1360</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1360</width>
     <height>768</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1360</width>
      <height>768</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1360</width>
      <height>768</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1360</width>
      <height>768</height>
     </size>
    </property>
    <widget class="QWidget" name="menu">
     <property name="minimumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_menu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="game">
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_game">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_time_game">
      <property name="geometry">
       <rect>
        <x>1139</x>
        <y>20</y>
        <width>141</width>
        <height>64</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_game">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>770</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>7680</width>
        <height>780</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_coins_game">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>20</y>
        <width>71</width>
        <height>64</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_picCoin_game">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_picTime_game">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>label_background_game</zorder>
     <zorder>graphicsView_game</zorder>
     <zorder>label_time_game</zorder>
     <zorder>label_coins_game</zorder>
     <zorder>label_picCoin_game</zorder>
     <zorder>label_picTime_game</zorder>
    </widget>
    <widget class="QWidget" name="leaderboard">
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_leaderboard">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="score_array">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>751</width>
        <height>471</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <property name="minimumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_settings">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_keys">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>401</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Key</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Action</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Z or Arrow Up</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Jump</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>D or Arrow Right</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>Run Right</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>Q or Arrow Left</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>Run Left</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>P</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_tips">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>520</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:14pt; font-weight:600;&quot;&gt;Tips : &lt;/span&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:14pt;&quot;&gt;While you're jumping you're faster !&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:14pt;&quot;&gt;It could be great to win some precious time.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="credits">
     <property name="minimumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_credits">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Developed by Hugo Bourreau

Thank's to all game testers and their ideas :
 - My parents (for their support too)
 - Arthur Porcher
 - Thomas Guillot
 - Thomas Brouard
 - Guillaume Jadeau
 - Marie Bompol
 - Etc..</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>50</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Game Background made by Arthur Porcher
Character photoshoped by Hugo Bourreau</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>90</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pictures from Nintendo : 
 =&gt; All right reserved
View more on their website :
 https://www.nintendo.fr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Don't forget to send me your highscore !</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="quit">
     <property name="minimumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_quit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pause">
     <property name="minimumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_pause">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="win">
     <widget class="QLabel" name="label_background_win">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="playername_lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>580</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lose">
     <property name="minimumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1360</width>
       <height>768</height>
      </size>
     </property>
     <widget class="QLabel" name="label_background_lose">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1360</width>
        <height>768</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1360</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
